pyproject.toml

```diff
@@ -88,3 +88,6 @@ ignore = [
     "RUF012", # Mutable class attributes should be annotated with `typing.ClassVar`
     "SIM105", # Use `contextlib.suppress(KeyError)` instead of `try`-`except`-`pass`
 ]
+
+[tool.black]
+line-length = 130
```

Comment:
As I said, my target is 118. But starting with 130 here is fine, then we can send a follow-up that changes it to 118. (FTR: I ran `black --line-length=118 --check --diff .|view -` locally against this PR, and that relative diff looks sensible).

Reply:
Thanks! Looks good IMHO.